client/src/game/GameLoop.js

```diff
@@ -1,8 +1,9 @@
 import drawPlanetside from "./drawPlanetside";
 import drawPlattforms from "./drawPlattforms";
+import drawRocket from "./drawRocket";
 import drawPlayer from "./drawPlayer";
 import letPlayerStandOnPlatform from "./letPlayerStandOnPlatform";
-import { PLATTFOMRHEIGHT, plattforms } from "./plattforms";
+import { PLATFOMRHEIGHT, platforms } from "./platforms";
```

Comment:
Platformheight? :)

Reply:
It's a static constant, so I wrote it in upper case ;)